share/man/man7/Makefile

```diff
@@ -42,7 +42,8 @@ MLINKS+= c.7 c95.7
 MLINKS+= c.7 c99.7
 MLINKS+= c.7 c11.7
 MLINKS+= c.7 c17.7
-MLINKS+= c.7 c2x.7
```

Comment:
We'll need an ObsoleteFiles.inc entry to remove the old c2x.7

Reply:
Thanks! I just pushed a commit, could you please take a look?